webdriver/tests/take_screenshot/user_prompts.py

```diff
@@ -2,63 +2,112 @@
 
```

Comment:
Same here for the missing `META` tag for a longer timeout.

Reply:
Same with the first comment you posted, I already see the `META` tag.